### PR TITLE
#381 add location field

### PR DESCRIPTION
**Pull request by jlwt90**
Hi, I added location to event form. 
#381 

Please let me know if there is any issue :)

### CODE REVIEW

File: src/pretix/control/templates/pretixcontrol/event/settings.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
             <legend>{% trans "General information" %}</legend>
             {% bootstrap_field form.name layout="horizontal" %}
             {% bootstrap_field form.slug layout="horizontal" %}
+            {% bootstrap_field form.location layout="horizontal" %}
```
Comment (raphaelm) [inline src/pretix/control/templates/pretixcontrol/event/settings.html]: I would recommend moving this two lines down below the date, as you did in the other form, as it is "less relevant".

**Comment by raphaelm** [comment]
Hey! :) Besides the one minor change I suggested, this looks fine! :)

**Comment by jlwt90** [comment]
@raphaelm Thanks! I didn't realise that the display orders were different in the two forms...
I added a new commit for this update :)

**Comment by raphaelm** [comment]
Perfect! Thank you very much!

**Pull request merged**